daily-programming/2024-05-20/main0520.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+using namespace std;
+long times(int x,int n){
+  long xx=1;
+  for(int i=0;i<n;i++){
+    xx=xx*x;
+  }
+  return xx;
+}
+int main(){
+  long n,a,b;
+  cin >> n >> a >> b;
+  if(1<=n && n<=times(10,18) && 0<=a && 0<=b && a+b<=times(10,18)){
+    long ab = a+b;
+    
+    long di = n/ab;

```

Comment:
『割った結果』を変数名とするならこっちの方が一般的だね
```suggestion
    long div = n / ab;
```